src/main/java/org/jabref/logic/git/GitHandler.java

```diff
@@ -28,9 +30,6 @@ public class GitHandler {
     static final Logger LOGGER = LoggerFactory.getLogger(GitHandler.class);
     final Path repositoryPath;
     final File repositoryPathAsFile;
-    String gitUsername = Optional.ofNullable(System.getenv("GIT_EMAIL")).orElse("");
-    String gitPassword = Optional.ofNullable(System.getenv("GIT_PW")).orElse("");
-    final CredentialsProvider credentialsProvider = new UsernamePasswordCredentialsProvider(gitUsername, gitPassword);
```

Comment:
This change will break the SLR feature, doesn't it?

Maybe, you can also rely that a user provided username and password in the environment variables `GIT_EMAIL` and `GIT_PW`.